source/Octopus.Tentacle.Tests.Integration/Support/IntegrationTest.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Threading;
+
+namespace Octopus.Tentacle.Tests.Integration.Support
+{
+    public abstract class IntegrationTest : IDisposable
```

Comment:
Consider adding the Serilog logger to this as well.

Reply:
I didn't want to have to copy that in for this PR. It will be a simple enough refactor when we finally get SSv2 PR in